README.md

```diff
@@ -1,3 +1,9 @@
 # Learning course Modern Statistics for Modern Biology @ statistical genetics lab
 
 The course is basically on R, but we try to implment algorithms in other computer languages
+
+## Resource
+[Modern Statistics for Modern Biology
```

Comment:
Thank you for adding the link!

Is it common to separate the link into 2 lines like
```md
[title
](url)
```
? if not, would you make it into 1 line?

Reply:
typo...